src/data/repositories/DiseaseOutbreakEventD2Repository.ts

```diff
@@ -44,14 +45,16 @@ export class DiseaseOutbreakEventD2Repository implements DiseaseOutbreakEventRep
         });
     }
 
-    save(diseaseOutbreak: DiseaseOutbreakEventBaseAttrs): FutureData<void> {
```

Comment:
please merge with Ana's feature/add-data-source-create-event branch and change the PR to merge into feature/add-data-source-create-event branch instead , so this change will not show up as a merge conflict